routes/truckRoutes.js

```diff
@@ -1 +1,105 @@
-// put your Express router code in here
\ No newline at end of file
+'use strict';
+// put your Express router code in here
+
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+
+var express = require('express');
+var foodTrucks = require('../trucks');
+
+var router = express.Router();
+
+var trucks = require('../trucks');
```

Comment:
I'm not sure what this code is for. You've already required in the trucks module and assigned it to foodTrucks. Each time you require it in, it's the same thing - you only need to do it once.